platform/ext/target/nordic_nrf/nrf5340/partition/flash_layout.h

```diff
@@ -213,7 +212,7 @@
 #define TFM_NV_COUNTERS_AREA_ADDR    FLASH_NV_COUNTERS_AREA_OFFSET
 #define TFM_NV_COUNTERS_AREA_SIZE    (0x18) /* 24 Bytes */
 #define TFM_NV_COUNTERS_SECTOR_ADDR  FLASH_NV_COUNTERS_AREA_OFFSET
-#define TFM_NV_COUNTERS_SECTOR_SIZE  FLASH_AREA_IMAGE_SECTOR_SIZE
+#define TFM_NV_COUNTERS_SECTOR_SIZE  4
```

Comment:
This macro does not seem to be used currently (it is only required to be defined) and I think it would be better to leave it as it was defined originally, for consistency with other platforms (it's hard to tell what value would be actually correct here).

Reply:
So, I should reset it to FLASH_AREA_IMAGE_SECTOR_SIZE?

Reply:
Done